content/en/blog/_posts/2020-12-04-gsod-2020-improving-api-reference-experience.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: "GSoD 2020: Improving the API Reference Experience"
+date: 2020-12-04
+slug: gsod-2020-improving-api-reference-experience
+---
+
+**Author**: [Philippe Martin](https://github.com/feloy)
+
+## Introduction
+
+[Google Season of Docs](https://developers.google.com/season-of-docs) brings open source organizations and technical writers together, to spend three months working closely on a specific documentation project.
+
+I was selected by the CNCF organization to work on the Kubernetes documentation, specifically on the subject of making the API Reference documentation more accessible.
+
+I'm a software developer, with a great interest in documentation systems. At the end of the 90's, I wanted to invest my time in the Linux community and started translating Linux-HOWTO documents. From one thing to another, I learned about documentation systems and finally wrote a Linux-HOWTO, to help document writers learn the language used at this time for writing documents, LinuxDoc/SGML.
+
+Shortly after, the DocBook language was adopted to write the Linux Documentation. I helped some writers rewrite their documents in this format, for example the Advanced Bash-Scripting Guide. I also worked on the GNU `makeinfo` program to add the DocBook output, making possible to transform *GNU Info* documentation into Docbook. 
+
+## Background
+
+The [Kubernetes documentation website](https://kubernetes.io/docs/home/) is built with Hugo from documentation written in Markdown format in the [website repository](https://github.com/kubernetes/website), using the [Docsy Hugo theme](https://www.docsy.dev/about/).
+
+The existing API reference documentation is a large HTML file generated from the Swagger specifications of the API, added to the content of the website.
```

Comment:
To be fair, it may be nice to say that you worked with [started working with] the current API reference.
Instead of focusing on "drawbacks", how about focusing on what you wanted to achieve [your goals].
Something like:
- build individual Markdown pages for the main Kubernetes resources
- reuse the website's assets and theme to build, integrate, and display the reference pages
- create reference pages that are more mobile friendly and accessible to search engines
- create reusable reference content